[wip] Dismiss component (#541)
microsoft/atlas-design: site/src/components/banner.md

```html-no-indent
<div class="banner is-loading">
    <div class="banner-content">
	    <p>Loading banner.</p>
    </div>
</div>
```

### Dismissing banner

> Banner can be made dismissable with the [`dismiss` subcomponent](./dismiss.md).

Author: Updated

	<p>Loading banner.</p>
</div>
```

### Dismissing banner

> Banner can be dismissed in conjunction usage with `dismiss` component. Read more about it [here](./dismiss.md).

Reviewer: I'd like to avoid links with generic text like "here." Perhaps something like this could work?

```suggestion
Banners can be made dismissable with [the `dismiss` component](./dismiss.md).
```

Author: Updated, thanks for recommendation :)

```html

> <div class="banner" id="example-banner-id" data-dismissable disappearing="slide-up">

Reviewer: See comment above - I don't think we should use `disappearing` as an attribute name. This should be made into a data attribute.

Author: Updated
